maven-extension/build.gradle.kts

```diff
@@ -60,3 +61,6 @@ tasks {
 }
 
 tasks.getByName("test").dependsOn("shadowJar")
+repositories {
+  mavenCentral()
+}
```

Comment:
is this really needed?

Reply:
Great catch, mistakenly added by my IDE (Intellij), I removed it